src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -48,7 +48,6 @@
 
 (rf/reg-event-fx :wallet/send-select-amount
  (fn [{:keys [db]} [{:keys [amount]}]]
-   (js/alert "Not implemented yet")
```

Comment:
FYI: re-frame events shouldn't have side-effects, though we have one famous exception in the project: we call log functions :)